lua/core/screen.lua

```diff
@@ -165,7 +165,7 @@ Screen.text_center = function(str) _norns.screen_text_center(str) end
 --- calculate width of text.
 -- uses currently selected font.
 -- @tparam string str : text to calculate width of
-Screen.extents = function(str) return _norns.screen_extents(str) end
+Screen.text_extents = function(str) return _norns.screen_extents(str) end
```

Comment:
Would it make sense to rename the "internal" function (`_norns.screen_extents()`) as well?